scripts/setup.sh

```diff
@@ -195,7 +195,7 @@ elif [ "$VAGRANT" = true ]; then
   # XVFB is required for headless testing in Vagrant
   sudo apt-get install xvfb chromium-browser
```

Comment:
OK. But how about the xvfb prefix env variable? Does that still need to be in setup.sh? If so, when does it get used outside the context of run_performance_tests.sh? (Concerned about people looking at setup.sh and drawing conclusions about this var when it's actually overwritten in another file.)